vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/SetContextSteps.java

```diff
@@ -66,66 +61,6 @@ public SetContextSteps(IUiContext uiContext, IWebDriverProvider webDriverProvide
         this.baseValidations = baseValidations;
     }
 
-    /**
-     * Set the context for further localization of elements to an <b>element</b> specified by the <b>name</b> and
-     * <b>state</b>
-     * @param name Any attribute or text value of the element
-     * @param state A state value of the element
-     * (<i>Possible values:</i> <b>ENABLED, DISABLED, SELECTED, NOT_SELECTED, VISIBLE, NOT_VISIBLE</b>)
-    */
-    @When("I change context to a [$state] element with the name '$name'")
-    public void changeContextToElementWithName(State state, String name)
-    {
-        resetContext();
-        WebElement element = baseValidations.assertIfElementExists(
-                String.format("An element with the name '%1$s'", name),
```

Comment:
known issues might be broken here, I propose to keep the commit, but postpone it till `0.3.0`